lib/omniauth/strategies/openid_connect.rb

```diff
@@ -47,8 +47,11 @@ class OpenIDConnect
       option :send_scope_to_token_endpoint, true
       option :client_auth_method
       option :post_logout_redirect_uri
+      option :uid_field, 'sub'
 
-      uid { user_info.sub }
+      uid do
+        user_info.send(options.uid_field.to_s)
```

Comment:
1. think it would be great to check if `user_info` respond to configured method
2. prefer to use `public_send` instead of `send`
3. please add some tests

Consider change your code to following:

```ruby
uid do
  next user_info.public_send(options.uid_field.to_s) if user_info.respond_to?(options.uid_field.to_s)
  user_info.sub
end
```